ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.0b11
+m_EditorVersion: 2017.2.0f1
```

Comment:
Did we also update the readme?

Reply:
Accidentally added the version txt.  I can revert it back unless we're going to be recommending an update soon.  If so I can also update the readme.

Reply:
Revert is easiest.  I think we should be using that version though.